prime-router/src/test/resources/test_data_files/Hl7_ORU-R01/EHT-20210316-0001.hl7

```diff
@@ -1 +1 @@
SFT|Lifepoint^L^^^^&2.16.840.1.113883.9.265&ISO^XX^^^Lifepoint|1.2.1|Lifepoint|Lifepoint Binary ID||20200601|
PID|1||LT-306^^^ELLUME LIMITED&2.16.840.1.113883.9.265&ISO^PT^|||||||UNK^UNKNOWN^HL70005|^^SANDY^UT^84093^^^^|||||||||||UNK^UNKNOWN^HL70189||||||||N|
ORC|RE||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO||CM|||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI||^^^^^^||||||||^^^^|^^^^^^|^^^^||||||||
OBR|1||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO|94558-4^LN^94558-4^SARS-CoV-2 (COVID-19) Ag [Presence]in Respiratory specimen by Rapid immunoassay^L|||202103160000|||||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI|^^^^^^|||||20210316000000-0500|||F||||||||||
OBX|1|CWE|94558-4^LN^260415000^260415000^L|1|260415000^Not detected^SCT|||N^Normal^HL70078|||F|||202103160000|||Ellume COVID-19 Home Test|151002609|202103160000||||ELLUME Limited|57 Didsbury St^^East Brisbane^QLD^4169|^|
NTE|1||Presumptive Negative Test for SARS-CoV-2 (no antigen detected)|
SPM|1|^1e837a04-7d87-4498-ac86-1476354ed257&ELLUME Limited&2.16.840.1.113883.9.265&ISO||^871810001^
SCT||||^871810001^SCT|||||||||202103160000|202103160000|||||202103160000
\ No newline at end of file
+MSH|^~\&#|LIFEPOINT^2.16.840.1.114222.4.5.1^ISO|ELLUME Limited^2.16.840.1.113883.9.265^ISO|WATERS_^2.16.840.1.113883.9.265^ISO|WATERS_^2.16.840.1.113883.9.265^ISO|20210316162100-0500||ORU^R01^ORU_R01|658195889000001-1e837a04-7d87-4498-ac86-1476354ed257|P|2.5.1||||NE|USA||||PHLabReport-NoAck^HL7^2.16.840.1.113883.9.11^ISO|||||
SFT|Lifepoint^L^^^^&2.16.840.1.113883.9.265&ISO^XX^^^Lifepoint|1.2.1|Lifepoint|Lifepoint Binary ID||20200601|
PID|1||LT-306^^^ELLUME LIMITED&2.16.840.1.113883.9.265&ISO^PT^||pablo|||||UNK^UNKNOWN^HL70005|^^SANDY^UT^84093^^^^|||||||||||UNK^UNKNOWN^HL70189||||||||N|
ORC|RE||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO||CM|||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI||^^^^^^||||||||test&test2&10^empty^NY^CA^|^^^^^^|^^^^||||||||
OBR|1||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO|94558-4^LN^94558-4^SARS-CoV-2 (COVID-19) Ag [Presence]in Respiratory specimen by Rapid immunoassay^L|||202103160000|||||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI|^^^^^^|||||20210316000000-0500|||F||||||||||
OBX|1|CWE|94558-4^LN^260415000^260415000^L|1|260415000^Not detected^SCT|||N^Normal^HL70078|||F|||202103160000|||Ellume COVID-19 Home Test|151002609|202103160000||||ELLUME Limited^^^^^^^^^test|57 Didsbury St^^East Brisbane^QLD^4169|^|
NTE|1||Presumptive Negative Test for SARS-CoV-2 (no antigen detected)|
SPM|1|^1e837a04-7d87-4498-ac86-1476354ed257&ELLUME Limited&2.16.840.1.113883.9.265&ISO||^871810001^
SCT||||^871810001^SCT|||||||||202103160000|202103160000|||||202103160000

```

Comment:
I will recommend not to change these sample files as they come directly from an entity.  Perhaps create a new generic HL7 and CSV pair with test data, or add unit tests for the specific function you are changing.

Reply:
The problem is that these test data files don't validate anymore given the new cardinality rule added when the tests get run with the 'package' task.
I'm not sure how can I stop the new Hl7Serializer's cardinality logic to be run only for the additional data sets and not for the existing ones.

Reply:
This is a problem since the data is from real samples.  This needs to be discussed.